build.gradle

```diff
@@ -60,6 +60,7 @@ configurations {
 }
 
 dependencies {
+    implementation 'org.postgresql:postgresql:42.2.23.jre7'

```

Comment:
Shouldn't this be shaded like the other libraries below it?

Reply:
Yes. I'll admit I didn't look in this file, I let my IDE add the dependency.